src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -93,8 +94,16 @@ export default class SaveInProgressIntro extends React.Component {
     return alert;
   }
 
+  getStartPage = () => {
+    const { form, route, location } = this.props;
+    const formData = form && form.data || {};
+    if (route && location) return getNextPagePath(route.pageList, formData, location.pathname);
+    return this.props.pageList[1].path;
+  }
```

Comment:
The idea here is to only calculate the first page if the first page is conditional.